src/operator-docs/transformation/bufferCount.ts

```diff
@@ -1,6 +1,87 @@
 import { OperatorDoc } from '../operator.model';
 
 export const bufferCount: OperatorDoc = {
-  'name': 'bufferCount',
-  'operatorType': 'transformation'
+  name: 'bufferCount',
+  operatorType: 'transformation',
+  signature: ` bufferCount(bufferSize: number, startBufferEvery: number): Observable<T[]>`,
```

Comment:
Furthermore we should genrally discuss whether using generics in method signature or not. @ashwin-sureshkumar, @ladyleet , @btroncone

Reply:
@ashwin-sureshkumar @JWO719 @ladyleet  Let's leave out per https://github.com/ReactiveX/rxjs-docs/pull/196#discussion_r157231328

Reply:
So please remove generic